src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/Marshallers/NativeThisToManagedThisMarshallerFactory.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Microsoft.Interop
+{
+    internal sealed record NativeThisInfo(ManagedTypeInfo TypeKeyType) : MarshallingInfo;
+
+    internal sealed class NativeToManagedThisMarshallerFactory : IMarshallingGeneratorFactory
```

Comment:
```suggestion
    internal sealed class NativeThisToManagedThisMarshallerFactory : IMarshallingGeneratorFactory
```

I assume this should match the name of the file.